.freeipa-pr-ci.yaml

```diff
@@ -62,3 +62,123 @@ jobs:
         template: *ci-master-f27
         timeout: 3600
         topology: *master_1repl
+
+  fedora-27/test_topologies:
+    requires: [fedora-27/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-27/build_url}'
+        test_suite: test_integration/test_topologies.py
+        template: *ci-master-f27
+        timeout: 3600
+        topology: *master_1repl
```

Comment:
It seems to me that the test_topologies test does not require any installation, so you could use a minimal topology with only one machine.